Externals.cfg

```diff
@@ -1,14 +1,14 @@
 # External sub-modules of global-workflow
 
 [FV3GFS]
-tag = GFS.v16.3.0
+tag = GFS.v16.3.1
 local_path = sorc/fv3gfs.fd
 repo_url = https://github.com/ufs-community/ufs-weather-model.git
 protocol = git
 required = True
 
 [GSI]
-tag = gfsda.v16.3.8
+tag = gfsda.v16.3.10
```

Comment:
curious why this is 16.3.10?  Shouldn't this be 16.3.11?
Not a blocker comment.

Reply:
When the GSI tag was cut for this upgrade we were still assuming this would go in with GFSv16.3.10.